src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json

```diff
@@ -29,7 +29,6 @@
       "commandName": "Project",
       "dotnetRunMessages": "true",
       "launchBrowser": true,
-      "hotReloadProfile": "blazorwasm",
```

Comment:
FYI @SteveSandersonMS / @danroth27

Reply:
Makes sense to hold off on enabling hot reload for Blazor WebAssembly by default until the experience is a bit more baked. Users can opt into trying it out if they want.

Reply:
Hold off on merging this.  I'm working on the bug

Reply:
https://github.com/dotnet/runtime/pull/50248 for main. I'll also open a backport for preview3